frontend/lib/src/components/elements/AlertElement/styled-components.tsx

```diff
@@ -17,12 +17,17 @@
 import styled from "@emotion/styled"
 import { StyledEmojiIcon } from "@streamlit/lib/src/components/shared/Icon/styled-components"
 
-export const StyledIconAlertContent = styled.div(({ theme }) => ({
+export const StyledAlertContent = styled.div(({ theme }) => ({
   display: "flex",
   gap: theme.spacing.sm,
+  width: "100%",
 
   [StyledEmojiIcon as any]: {
     position: "relative",
     top: "2px",
   },
+
+  " code": {
```

Comment:
It probably needs to be more specific to only cover code uses inside a multiline code block, e.g. something like `. stCodeBlock code`